client/src/main/java/tds/session/Session.java

```diff
@@ -42,4 +53,129 @@ public int getType() {
     public void setType(int type) {
         this.type = type;
     }
+
+    /**
+     * @return The time when the {@link Session} was started.
+     * <p>
+     *     Should handle date and time in UTC
+     * </p>
+     */
+    public Timestamp getDateBegin() {
```

Comment:
Let's make these Instant instead of exposing sql.Timestamp.  I realize you'll have to do some extra mapping in the DB layer